apps/api/src/certificate/http-schemas/create-certificate.schema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const CreateCertificateResponseSchema = z.object({
```

Comment:
can we wrap it in `data` as a common approach?

Reply:
Oh yea true